ansible/roles/analytics-spark-provision/templates/spark-env.j2

```diff
@@ -72,8 +72,8 @@ export SPARK_EXECUTOR_MEMORY={{ spark.executor.memory }}
 export SPARK_PUBLIC_DNS="{{ spark.public_dns }}"
 export reports_storage_key={{sunbird_private_storage_account_name}}
 export reports_storage_secret={{sunbird_private_storage_account_key}}
-export azure_storage_key={{sunbird_private_storage_account_name}}
```

Comment:
Please don't change the internal variables

Reply:
`sunbird_private_storage_account_key` and  `sunbird_public_storage_account_name` is internal variable ? is it refered to common variable or not ?